src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,84 @@
+$toggle-track-length: rem(46px);
+$toggle-track-height: rem(24px);
+$toggle-thumb-size: rem(20px);
+
+.Toggle {
+  display: flex;
+  align-items: center;
+
+  &.disabled {
+    opacity: 0.8;
+  }
+}
+
+.ToggleTrack {
+  width: $toggle-track-length;
+  height: $toggle-track-height;
+  border-radius: border-radius(extra-large);
+  border: border-width() solid color('sky', 'dark');
+  background-color: color('sky');
+  transition: background-color duration(slow), border duration(slow);
+  -webkit-tap-highlight-color: transparent;
+}
+
+.ToggleThumb {
+  position: absolute;
+  top: rem(2px);
+  left: rem(2px);
+  width: $toggle-thumb-size;
+  height: $toggle-thumb-size;
+  border-radius: 50%;
+  transition: left duration(slow) easing();
+  background-color: color('white');
+}
+
+.ToggleChecked {
+  .ToggleTrack {
+    background-color: color('indigo');
+    border: border-width() solid color('indigo', 'dark');
+    outline: none;
+  }
+
+  .ToggleThumb {
+    left: $toggle-track-height;
+  }
+}
+
+.ToggleDisabled {
+  cursor: not-allowed;
```

Comment:
Hmm, why `not-allowed` here as opposed to another cursor property? What does this give us?

Reply:
The only other one that makes sense to me here is `none`

Reply:
It adds 🚫 to the cursor.

![screen shot 2018-11-13 at 3 29 43 pm](https://user-images.githubusercontent.com/6239480/48449953-209bff00-e759-11e8-944e-1f1ecfd56fd9.jpg)